docs/tooling/IDEs/promptsandbox.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 22
+---
+
+# PromptSandbox.io
+
+[PromptSandbox.io](https://PromptSandbox.io) is a node-based visual programming tool to create workflows with OpenAI APIs
+
+[Repo](https://github.com/eg9y/promptsandbox.io/)
```

Comment:
Use full sentences here (e.g. the repo can be found here), perhaps describe it slightly more.

Reply:
added full sentences